.github/workflows/publish-release.yaml

```diff
@@ -0,0 +1,111 @@
+name: Publish release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish_release:
+    name: Publish release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      id-token: write
+    # Note: `prepare-release.yaml` sets this commit message
+    if: ${{ contains(github.event.head_commit.message, 'release @tutorialkit packages') }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: ./.github/actions/setup-and-build
+
+      # Sets steps.resolve-release-version.outputs.version
+      - uses: ./.github/actions/resolve-release-version
+        id: resolve-release-version
+
+      - name: Publish to npm
+        run: >
+          pnpm --recursive
+          --filter "@tutorialkit/*"
+          --filter create-tutorial
+          exec pnpm publish --provenance
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Create and push git tag
+        run: |
+          git config --global user.name 'github-actions'
+          git config --global user.email 'github-actions@users.noreply.github.com'
+          git tag v${{ steps.resolve-release-version.outputs.version }}
+          git push origin v${{ steps.resolve-release-version.outputs.version }}
+
+  prepare_cli_release:
+    name: Prepare release for CLI
+    needs: [publish_release]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: ./.github/actions/setup-and-build
+
+      # Sets steps.resolve-release-version.outputs.version
+      - uses: ./.github/actions/resolve-release-version
+        id: resolve-release-version
+
+      - name: Bump version
+        run: >
+          pnpm --recursive
+          --filter tutorialkit
+          exec npm version --no-git-tag-version --allow-same-version ${{ steps.resolve-release-version.outputs.version }}
+
+      - name: Generage changelog
+        run: |
+          pnpm run changelog
+          git add ./packages/cli ./packages/template
+          git checkout ./packages
+          git checkout CHANGELOG.md
```

Comment:
So this is the step I would remove as my guess is that I think it won't work because the tag for the latest version was already pushed.